swagger/llc_initial.json

```diff
@@ -3,14 +3,27 @@
   "info": {
     "version": "1.0.0",
     "title": "LLCClient",
-    "description": "LLC Swagger, this is the initial swager a service could do"
+    "description": "LLC Swagger, this is the initial swagger a service could do"
   },
   "host": "localhost:3000",
   "schemes": ["http"],
   "produces": ["application/json"],
   "consumes": ["application/json"],
   "paths": {
-    "/servicedriven/parameters": {
+    "/serviceDriven/parameters": {
+      "head": {
+        "operationId": "params_noParams",

```

Comment:
can we name it "headNoParams"? Just in case we want more no params tests later

Reply:
Sure! :)